echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/monitor/TriggerMonitor.java

```diff
@@ -74,12 +74,17 @@ private void triggerMatchingPipelines(T event) {
     try {
       List<Pipeline> matchingPipelines = eventHandler.getMatchingPipelines(event, pipelineCache);
       matchingPipelines.stream()
+          .filter(p -> p.getErrorMessage() == null)
```

Comment:
Any thoughts on whether we should also allow the empty string through this filter?  (And exclude it from the filter below?)

Reply:
was contemplating that myself and don't recall why i picked one over the other :)
but it's prob safer to `isNullOrEmpty()`it